modules/persistent-storage-csi-vsphere-migrating-cns-vols-between-datastores.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-vsphere-migrating-cns-vols-between-datastores_{context}"]
+= Migrating CNS volumes between datastores for vSphere
+
+If you are running out of space in your current datastore, or want to move to a more performant datastore, you can migrate VMware vSphere Cloud Native Storage (CNS) volumes between datastores. This applies to both attached and detached volumes.
+
+.Limitations
```

Comment:
I think we have a limitation with RWX volumes.
Also IIRC we wanted to add a note (be cautious) about encrypted volumes and topology
@gnufied

Reply:
Yes, RWX not supported. I've added this limitation. Encrypted vols do not have any restriction according to Hemant. Topology limitation is covered in linked content.